server/routes/cloudbees/cloudbees_routes.py

```
@@ -224,6 +226,197 @@ def disconnect(user_id):
+        return jsonify({"error": "Username is required"}), 400
+
+    logger.info("[CLOUDBEES] Connecting platform for user %s (OC: %s, mode: %s)", user_id, oc_url, "pat" if is_pat_mode else "basic")
+
+    # Validate OC connection
+    from connectors.cloudbees_connector.oc_client import CloudBeesOCClient
+
+    if is_pat_mode:
+        oc_client = CloudBeesOCClient(base_url=oc_url, username="", api_token=api_token, auth_mode="bearer")
+    else:
+        oc_client = CloudBeesOCClient(base_url=oc_url, username=username, api_token=api_token)
+    success, server_data, error = oc_client.get_server_info()
```

Ruff (0.15.15)

[warning] Unpacked variable `server_data` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)